datalab_sdk/models.py

```diff
@@ -64,6 +64,7 @@ class ConversionResult:
     markdown: Optional[str] = None
     html: Optional[str] = None
     json: Optional[Dict[str, Any]] = None
+    extraction_schema_json: Optional[str] = None
```

Comment:
The API opts to return a string to be safe in case there's weird formatting / character escape issues (i think more common when extracting latex) and it fails to properly parse to json - we still want the raw extraction value instead of fully failing